### DISCUSSION TITLE
Relevance of version 1.7.10 and 1.8.8. Requesting further support for these versions

### DISCUSSION THREAD

**Discussion by microwin7** [Zrips/CMI#5884] [General]
Всем Привет. 
Хочу начать с того наверное, как я купил этот плагин. На проекте где я работаю, у нас есть различные сервера, 1.7.10, 1.8.8 и 1.12.2
В частности 1.7.10 только модовые, когда я покупал, была надежда что плагин будет хорошо работать на всех версиях, как и заявлено. Но тогда же я столкнулся с проблемой, которую я расписал здесь: https://github.com/CrucibleMC/Crucible/issues/77
Долгое время никуда не обращался, только пробовал сам устранить эту проблему в Thermos ядре, но неудачно. Скажу сразу, по обращению, разработчик ядра сказал что исправил эту проблему, которую создали ещё при написании Cauldron ядра.
Когда покупал, это был где-то год назад, версия CMI работала на 1.7.10 Spigot, но не работала на 1.7.10 Thermos, тогда я утратил все надежды. Недавно узнав про ядро Crucible, уже смог обратиться за помощью, так же ожидаю и поддержки с вашей стороны, если данная дискуссия оправдает себя, буду отправлять ошибки с 1.7.10 версии. 
Мне говорят что 1.7.10 более не популярна, но bstats на Thermos 1.7.10 я так полагаю вообще не работает, ибо там был http://mcstats.org/plugin/Thermos который не работает, значит и статистики с этой версии вообще нет.
Мной был сделан опрос из создателей весьма крупных проектов у нас в России, выборка небольшая, так как у нас закрытый чат, с отбором, по большей степени по репутации.
![image](https://user-images.githubusercontent.com/12544425/125876721-40cf9c6c-d2fb-4cad-aff1-00032213a36c.png)

Так же дальнейшее описание можно прочитать уже в самом канале CMI
https://discord.com/channels/452792793631555594/526402563847880725/865384971182932029

Надеюсь вы меня услышите и дадите положительный ответ, что поддержка будет продолжаться

Google Translate:
```
Hello everyone.
I want to start by probably telling you how I bought this plugin. On the project where I work, we have different servers, 1.7.10, 1.8.8 and 1.12.2
In particular 1.7.10 is only mod, when I bought it, it was hoped that the plugin would work well on all versions, as stated. But then I ran into a problem that I described here: https://github.com/CrucibleMC/Crucible/issues/77
For a long time I did not go anywhere, I just tried to fix this problem in the Thermos core myself, but it was unsuccessful. I will say right away that upon request, the kernel developer said that he had fixed this problem, which was created when the Cauldron kernel was written.
When I bought it, it was about a year ago, the CMI version worked on 1.7.10 Spigot, but did not work on 1.7.10 Thermos, then I lost all hope. Having recently learned about the Crucible kernel, I was already able to ask for help, I also expect support from your side, if this discussion justifies itself, I will send bugs from version 1.7.10.
They tell me that 1.7.10 is no longer popular, but I suppose bstats does not work at all on Thermos 1.7.10, because there was http://mcstats.org/plugin/Thermos which does not work, which means that there are no statistics from this version at all ...
I made a survey of the creators of very large projects in Russia, the sample is small, since we have a closed chat, with a selection, mostly by reputation.
! [image] (https://user-images.githubusercontent.com/12544425/125876721-40cf9c6c-d2fb-4cad-aff1-00032213a36c.png)

You can also read further description in the CMI channel itself.
https://discord.com/channels/452792793631555594/526402563847880725/865384971182932029

I hope you hear me and give a positive answer that support will continue
```

**Comment by Zrips** [comment]
For time being we are keeping support for those versions, but this cant be for ever as new versions come out new things get implemented which gets used as features while in old servers this will not work and majority of testing is done in newest versions. With time, this support will be more and more limited and we will have to drop it to simply plugin maintenance. For now its all good.

**Reply by microwin7** [reply]
Then I will try in the near future to collect the problems that exist, in the hope of eliminating them, at least there will be a more or less stable version of the plugin running on these servers
